packages/taco/test/conditions/compound-condition.test.ts

```diff
@@ -135,7 +135,7 @@ describe('validation', () => {
   const multichainCondition: CompoundConditionProps = {
     conditionType: CompoundConditionType,
     operator: 'and',
-    operands: SUPPORTED_CHAIN_IDS.map((chain) => ({
+    operands: [ChainId.SEPOLIA, ChainId.AMOY].map((chain) => ({
```

Comment:
To keep the same test:

```suggestion
    operands: [ChainId.ETHEREUM_MAINNET, ChainId.POLYGON, ChainId.SEPOLIA, ChainId.AMOY].map((chain) => ({
```